src/client.cpp

```diff
@@ -862,50 +845,88 @@ void CClient::OnClientIDReceived ( int iChanID )
     emit ClientIDReceived ( iChanID );
 }
 
-void CClient::Start()
+bool CClient::Connect ( QString strServerAddress, QString strServerName )
```

Comment:
I believe we may want to still have the split into two functions for modularity, but provide an API for a direct connection.